src/validate.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes */
```

Comment:
- [ ]  [optional] @francis3400 its not good practise to disable linters, always try and solve them.